src/encoded/upgrade/genetic_modification.py

```diff
@@ -294,3 +294,13 @@ def genetic_modification_9_10(value, system):
     if 'donor' in value:
         value['introduced_elements_donor'] = value['donor']
         value.pop('donor')
+
+
+@upgrade_step('genetic_modification', '10', '11')
+def genetic_modification_10_11(value, system):
+    # https://encodedcc.atlassian.net/browse/ENCD-5968
+    notes = value.get('notes', '')
+    if value.get('introduced_elements', '') == 'gRNAs and CRISPR machinery':
+        if 'guide_type' not in value:
+            value['guide_type'] = 'sgRNA'
+            value['notes'] = (notes + ' guide_type on this GM was defaulted to sgRNA in an upgrade.').strip()
```

Comment:
not sure the .strip is needed in this addition, but it cannot hurt.

Reply:
It should remove the space at the beginning of the string if 'notes' is empty before the upgrade